README.md

```diff
@@ -95,6 +95,25 @@ call plug#end()
 
 lua require("noice").setup()
 
+```
+</details>
+
+<details><summary>lazy.nvim</summary>
+
+```lua
+-- lazy.nvim
+{
+  "folke/noice.nvim",
+  config = function()
```

Comment:
you could also use `opts`

Reply:
Should definitely use opts! Especially because this function completely ignores any options set anywhere. At the least it should be 
```suggestion
  config = function(_, opts)
```
and then use that `opts` table with whatever changes, but really no reason to if you just call `setup`..